### DISCUSSION TITLE
Building page is problem from HTML CSS codes

### DISCUSSION THREAD

**Discussion by Subha-Paul** [community/community#108477] [Programming Help]
### Select Topic Area

General

### Body

Hi, I am beginner and in the process of learning. I uploaded drag and drop file of contains HTML CSS codes. Whenever I want to make page of it just shows readme in the sitelink. here https://github.com/Subha-Paul is my profile go and check codes are there. But help me hwo to show the webpage hosting instead showing readme

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Subha-Paul ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Comment by davea42** [comment]
Well, when I click your project I see the text of your README.md .  That is what github does
(and your README.md is nicely written).

What you want is a web server with your jpg, html, and css.  That is not what github supplies or provides.

You can create one on your personal machine (details depend on the OS and system involved)
and  copy the parts to the base directory applicable (where that is depends on your OS
and system).  

Hope this makes sense and helps a little.
I hesitate to guess what short of machine (os/system) is, so I am not sure what more would be relevant.

You will get there, what you have done shows definite promise.  
David Anderson

**Comment by ghost** [comment]
You can use github to host your project, but the index.html and other website files should be in root directory for it to work(easy way).

**Reply by ghost** [reply]
[Link](https://youtu.be/OltY8JIaP-4?si=qY-D_MCEWepCd_cv) follow this tutorial
